insecure/corruptible/network_common_test.go

```diff
@@ -0,0 +1,148 @@
+package corruptible
+
+import (
+	"fmt"
+	"github.com/onflow/flow-go/insecure"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/libp2p/message"
+	"github.com/onflow/flow-go/network/channels"
+	"github.com/onflow/flow-go/network/codec/cbor"
+	"github.com/onflow/flow-go/network/mocknetwork"
+	"github.com/onflow/flow-go/utils/unittest"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"sync"
+	"testing"
+	"time"
+)
+
+// This test file covers corruptible network tests that are not ingress or egress specific, including error conditions.
+
+// TestEngineClosingChannel evaluates that corruptible network closes the channel whenever the corresponding
+// engine of that channel attempts on closing it.
+func TestEngineClosingChannel(t *testing.T) {
+	corruptibleNetwork, adapter := corruptibleNetworkFixture(t, unittest.Logger())
+	channel := channels.Channel("test-channel")
```

Comment:
nit: use channels.TestNetworkChannel

```suggestion
	channel := channels.TestNetworkChannel
```

Reply:
replaced all occurrences https://github.com/onflow/flow-go/pull/2811/commits/10c85a68a545fc9d8d1341d9bd7ee7364e7b0fd0